app/src/test/kotlin/com/intive/picover/parties/repository/PartiesRepositoryTest.kt

```diff
@@ -25,7 +26,7 @@ class PartiesRepositoryTest : ShouldSpec(
 		val tested = PartiesRepository(firebaseDatabase)
 
 		beforeSpec {
-			mockkStatic("com.google.firebase.database.ktx.DatabaseKt")
```

Comment:
Always try to avoid referencing classes of functions by strings whenever possible